experimentation/build.gradle

```diff
@@ -1,13 +1,14 @@
 plugins {
-    id 'com.android.library'
-    id 'org.jetbrains.kotlin.android'
+    id "com.android.library"
+    id "org.jetbrains.kotlin.android"
+    id "org.jetbrains.kotlin.kapt"
```

Comment:
If I'm not mistaken, as we don't use Dagger here anymore, we can remove `kapt` plugin.

```suggestion
```

Reply:
Ops! Forgot about this one 😅 
Removed, thanks!